sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CollectionExpressionsSuite.scala

```diff
@@ -2752,4 +2752,18 @@ class CollectionExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper
     )
 
   }
+
```

Comment:
There is an existing test to verify insert of a null value: https://github.com/apache/spark/blob/26f2026c8bb11064695ebf87b198a7dedf5bb73e/sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CollectionExpressionsSuite.scala#L2332

It succeeds only because the input array (`a1`) already has `containsNull` set to true:

https://github.com/apache/spark/blob/26f2026c8bb11064695ebf87b198a7dedf5bb73e/sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CollectionExpressionsSuite.scala#L46